sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/SessionWindow.scala

```diff
@@ -68,11 +68,29 @@ case class SessionWindow(timeColumn: Expression, gapDuration: Expression) extend
   with Unevaluable
   with NonSQLExpression {
 
+  private def inputTypeOnTimeColumn: AbstractDataType = {
```

Comment:
(Oh I missed that the change from rule side has already merged in as a part of introduction of `window_time`.)

Reply:
Nope. Just for future visibility.